### ISSUE TITLE
Suggestion - Eye room spells

### ISSUE THREAD

**Issue by maxsupermanhd** [TwoAbove/noita-tools#340]
**Describe the solution you'd like**
Panel or a list or something to generate eye room spells for given world

**Comment by TwoAbove** [comment]
Oh, nice one! Thanks.

**Comment by ViZeShadowZ** [comment]
adding to this, spells for The Works shops too

**Comment by TwoAbove** [comment]
Coming along bit by bit. Right now Noitool has the meditation cube and eye portal wands.

**Comment by maxsupermanhd** [comment]
Any news on this?

**Comment by TwoAbove** [comment]
No, I'm stuck with spell coordinates - my reverse-engineering of the map generator is not accurate in the eye spell room, so I'm debugging that part first. I unfortunately have no timelines for it

**Comment by TwoAbove** [comment]
Update - I can't seem to get the spawn positions accurately, so I would be thankful if you will be able to get those for me. Ideally, it would be - seed + screenshot of the room + positions

**Comment by TwoAbove** [comment]
Update - I can't seem to get the spawn positions accurately, so I would be thankful if you will be able to get those for me. Ideally, it would be - seed + screenshot of the room + positions

**Comment by maxsupermanhd** [comment]
Well, I am in fact a coder and a modder of a few games but I am yet to mod noita and get code running and hooks in it so it might have to wait for some time. I am not familiar with anything regarding modding in noita. If it is at all possible can you link me to some place to start injecting code and directing away calls inside noita? Are there annotations or community built symbols available?

Also I am in official Noita discord, do people dig that deep on there that I can ask?

**Comment by maxsupermanhd** [comment]
Oh and also a big problem for me is that Noita does not ship native Linux build and I don't have Windows system at all in reach (aside from like vms or similar) so it might complicate things...

**Comment by TwoAbove** [comment]
I actually run Noita on Linux through proton, but I don't have that laptop with me right now. 

I think asking on the official noita discord might be a better option and maybe someone can quickly whip up the coords in the modding channels
